src/js/page/utils.js

```diff
@@ -89,3 +91,42 @@ export function trackFocusMethod() {
     focusMethod = 'mouse';
   }, true);
 }
+export const copyUtils = {
+  _text: '',
+  _pre: document.createElement('pre'),
+  setCopyText(text) {
+    this._text = text;
+  },
+  copyText() {
+    if (!this._text) return false;
+  
+    this._pre.textContent = this._text;
+    document.body.appendChild(this._pre);
+    getSelection().removeAllRanges();
+
+    const range = document.createRange();
+    range.selectNode(this._pre);
+
+    window.getSelection().addRange(range);
+
+    document.execCommand('copy');
+    getSelection().removeAllRanges();
+    document.body.removeChild(this._pre);
+    
+    return true;
+  }
+};
```

Comment:
Could this just be `copyText(text)`? Instead of calling `setCopyText()`, components would just store the text for the  SVG (they probably already do).